README.md

```diff
@@ -30,6 +30,7 @@ We use three metrics in the SRS benchmark to evaluate how well these algorithms
 
 ### Algorithms
 
+- FSRS v1 and v2: alpha versions of FSRS that weren't publicly available.
```

Comment:
Nope. They are public: https://www.reddit.com/r/Anki/comments/xhfl0p/new_progress_in_implementing_the_custom_algorithm/

Reply:
How would you phrase this then?

Reply:
They were available to be used for custom scheduling. I provided the script and optimizer: 

https://github.com/open-spaced-repetition/fsrs4anki/blob/v1.0.0-alpha/fsrs4anki_scheduler.js

https://github.com/open-spaced-repetition/fsrs4anki/blob/v1.0.0-alpha/fsrs4anki_optimizer.ipynb

Reply:
Then we need to change the description of FSRS v3, since it's not actually "the first version of the FSRS algorithm that people actually used."

Reply:
FSRS v3 is the first version support FSRS helper add-on: https://www.reddit.com/r/Anki/comments/xw6jq3/big_update_in_fsrs4anki_v300/